talktorials/11_online_apis/11b_online_docking.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<p>As already commented in 11a a bit - I suggest to decide on one formatting style for the final version of T11. Maybe also link the references to the topic they are describing as suggested in 11a?</p><p> </p><p>Here again the summary of references for methods used here (taken from below, this time not reformatted since I don't know what format style you will use in the end):</p><p> </p><p><code>* DoGSiteScorer</code></p><p><code>&nbsp;* J. Chem. Inf. Model. 2010, 50, 11, 2041-2052, https://doi.org/10.1021/ci100241y</code></p><p><code>&nbsp;* Bioinformatics 2012, 28, 15, 2074–2075, https://doi.org/10.1093/bioinformatics/bts310</code></p><p><code>* SwissDock</code></p><p><code>&nbsp;* Nucleic Acids Res. 2011 Jul;39(Web Server issue):W270-7. doi: 10.1093/nar/gkr366. https://academic.oup.com/nar/article/39/suppl_2/W270/2506492</code></p><p><code>&nbsp;* J Comput Chem. 2011 Jul 30;32(10):2149-59. doi: 10.1002/jcc.21797. https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.21797</code></p><p><code>* OPAL webservices</code></p><p><code>&nbsp;* Nucleic Acids Res. 2010 Jul;38(Web Server issue):W724-31. doi: 10.1093/nar/gkq503 https://academic.oup.com/nar/article/38/suppl_2/W724/1122840</code></p><p><code>&nbsp;* Opal: Simple Web Services Wrappers for Scientific Applications http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.533.7960&amp;rep=rep1&amp;type=pdf</code></p><p><code>* AutoDock Vina </code></p><p><code>&nbsp;* J Comput Chem. 2010 Jan 30; 31(2): 455–461. doi: 10.1002/jcc.21334 https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.21334</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>